api-extractor/src/PrettyPrinter.ts

```diff
@@ -10,23 +10,36 @@ export default class PrettyPrinter {
     * Used for debugging only.  This dumps the TypeScript Compiler's abstract syntax tree.
     */
   public static dumpTree(node: ts.Node, indent: string = ''): void {
-    if (node.kind === ts.SyntaxKind.FromKeyword) {
-      console.log('**');
-    }
+    try {
+      const jsdoc: string = TypeScriptHelpers.getJsDocComments(node, console.log);
+      if (jsdoc && jsdoc.trim()) {
```

Comment:
Delete this COMMENT= code; it was just for testing.  Also delete the console.log('**') thing above.